src/commands/build.ts

```diff
@@ -15,9 +15,13 @@
 // Only import type definitions here, otherwise this line will be included in
 // the JS output, triggering  the entire build library & its dependencies to
 // be loaded and parsed.
+import * as delTypeOnly from 'del';
+import * as path from 'path';
```

Comment:
I think the guideline should be to defer heavyweight imports before running commands, rather than not loading anything at all. That said, I reverted the `path` import here.